worker/worker.go

```diff
@@ -194,7 +252,7 @@ func (r *DefaultWorker) openStepLogs(mapper *FileMapper, s *stepWorker, d *tes.E
 	if d.Stdin != "" {
 		s.Command.Stdin, err = mapper.OpenHostFile(d.Stdin)
 		if err != nil {
-			s.Event.Error("Couldn't prepare log files", err)
+			err := s.Event.Error("Couldn't prepare log files", err)
```

Comment:
This change causes the wrong error to be returned. 

In general, it might be worth considering an approach where you just log all event writer errors rather than handle them throughout the codebase. In this PR, the new error handling seems inconsistent - errors are logged, returned, ignored or even involved in a panic.

Reply:
IIRC (and it has been a looong time) we intentionally ignored these sorts of errors. Why? I have no idea.

Reply:
I think it was because we generally used the `MultiWriter` with the `Logger` writer ensuring that the events were recorded somewhere.

Reply:
The Writer is actually already wrapped with one that logs all errors that occur:

https://github.com/ohsu-comp-bio/funnel/blob/226edc9f5c8e4e85b7c3c7c39f41c5d32e886c3f/cmd/worker/run.go#L58

Reply:
Ah, that makes sense! Changing `err :=` to `_ =` to satisfy the errcheck linter:

```diff
- err := s.Event.Error("Couldn't prepare log files", err)
+ _ = s.Event.Error("Couldn't prepare log files", err)
```

Original linting error:
```sh
➜  funnel git:(fix/lint) ✗ golangci-lint run ./... -D unused -e "deprecated"
worker/worker.go:255:17: Error return value of `s.Event.Error` is not checked (errcheck)
                        s.Event.Error("Couldn't prepare log files", err)
```